tests/unit/states/archive_test.py

```diff
@@ -72,15 +67,9 @@ def test_extracted_tar(self):
                                                    'file.file_exists': mock_false,
                                                    'file.makedirs': mock_true,
                                                    'cmd.run_all': mock_run}):
-                    if HAS_PWD:
-                        running_as = pwd.getpwuid(os.getuid()).pw_name
```

Comment:
Point taken. But it did happened to me. I'll see if I remember tomorrow morning to try it again.